src/cards/ScenarioNode/ScenarioNode.js

```diff
@@ -248,7 +252,9 @@ export const ScenarioNode = ({
         {getDetailedStatus()}
         <Typography className={classes.cardLabel}>{getDatasetsLabel()}</Typography>
         <Typography>
-          <span className={classes.datasets}>{DatasetUtils.getDatasetNames(datasets, scenario.datasetList)}</span>
+          <span data-cy="scenario-dataset" className={classes.datasets}>
```

Comment:
This should probably be `scenario-datasets` (plural), because it could contain several datasets (from the component point of view, at least; not sure if we can provide several datasets to a scenario from the webapp, maybe with the API)